5333/src/main/include/BoxIntake.h

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <frc/Servo.h>
+
+#include "strategy/Strategy.h"
+#include "devices/DeployableDevice.h"
+#include "CurtinControllers.h"
+#include "Toggle.h"
+
+using BoxIntakeState = curtinfrc::devices::DeployableDeviceState;
+
+struct BoxIntakeConfig : public curtinfrc::devices::DeployableDeviceConfig {
+  frc::Servo &servo;
```

Comment:
What part of the BoxIntake has a servo?

Reply:
Oh, I forgot to remove that. I will fix that this afternoon.